frontend/src/modules/quickstart/components/item/quickstart-guide-item-content.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <article class="pt-2 pl-9">
+    <img :alt="props.guide.title" :src="`/images/quickstart/${props.guide.key}.jpg`" class="w-full">
+
+    <div class="flex items-center justify-center pt-3">
+      <a
+        v-if="props.guide.learnMoreLink"
+        :href="props.guide.learnMoreLink"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <el-button
+          class="btn btn--transparent btn--md mb-4 leading-5 mx-2"
+        >
+          <i class="ri-book-open-line mr-2" />Learn more
+        </el-button>
+      </a>
+
+      <router-link :to="props.guide.buttonLink">
+        <el-button
+          v-if="props.guide.buttonText"
+          class="btn btn--primary btn--md mb-4 leading-5 mx-2"
+          @click="trackBtnClick(props.guide.key)"
```

Comment:
Can we also add tracking for when they expand one of the guides?

Reply:
done